Shell/main.cpp

```diff
@@ -191,33 +194,57 @@ static int sh_exit(int, const char**)
 
 static int sh_export(int argc, const char** argv)
 {
-    if (argc == 1) {
+    Vector<const char*> vars;
+
+    Core::ArgsParser parser;
+    parser.add_positional_argument(vars, "list of variable[=value]'s", "values", Core::ArgsParser::Required::No);
+
+    if (!parser.parse(argc, const_cast<char**>(argv), false))
+        return 1;
+
+    if (vars.size() == 0) {
         for (int i = 0; environ[i]; ++i)
             puts(environ[i]);
         return 0;
     }
-    auto parts = String(argv[1]).split('=');
-    if (parts.size() != 2) {
-        fprintf(stderr, "usage: export variable=value\n");
-        return 1;
-    }
 
-    int setenv_return = setenv(parts[0].characters(), parts[1].characters(), 1);
+    int return_value = 0;
+
+    for (auto& value : vars) {
+        auto parts = String { value }.split_limit('=', 2);
+
+        if (parts.size() == 1) {
+            parts.append("");
+        }
+
+        int setenv_return = setenv(parts[0].characters(), parts[1].characters(), 1);
+
+        if (setenv_return != 0) {
+            perror("setenv");
+            return_value = 1;
+            break;
+        }
 
-    if (setenv_return == 0 && parts[0] == "PATH")
-        cache_path();
+        if (parts[0] == "PATH")
+            cache_path();
+    }
 
-    return setenv_return;
+    return return_value;
 }
 
 static int sh_unset(int argc, const char** argv)
 {
-    if (argc != 2) {
-        fprintf(stderr, "usage: unset variable\n");
+    Vector<const char*> vars;
+
+    Core::ArgsParser parser;
+    parser.add_positional_argument(vars, "list of variables", "variables", Core::ArgsParser::Required::Yes);
```

Comment:
Here and below: the convention is to start the help text with capital letter.

Reply:
Noted.